.env.example

```diff
@@ -11,7 +11,7 @@
 
 # Prisma
 # https://www.prisma.io/docs/reference/database-reference/connection-urls#env
-DATABASE_URL="file:./db.sqlite"
+DATABASE_URL="postgresql://Thermometer_Ninja:HotCocoa42@localhost:5432/Thermal_Rising"
```

Comment:
Is this the address of the local docker db?

Reply:
Yeah those were the environment variables I gave to the blank postgreSQL image for setup
```
    ports:
      - '5432:5432'
    environment:
      POSTGRES_USER: Thermometer_Ninja
      POSTGRES_PASSWORD: HotCocoa42
      POSTGRES_DB: Thermal_Rising
```